### PR TITLE
Added basic command to show effective input value

### PR DESCRIPTION
**Pull request by pigmej**
It shows effective single input value.
```
vagrant@solar-dev:/vagrant$ solar resource effective_iv cinder_puppet db_host
- cinder_puppet::db_host
- - mariadb_service1::ip
  - - node0::ip

--------------------
10.0.0.3
--------------------
```

### CODE REVIEW

File: solar/solar/cli/resource.py [hunk 1]
```diff
@@ -63,33 +63,45 @@ def action(dry_run_mapping, dry_run, action, resource):
...
 
-        bi = i.backtrack_value_emitter(level=1)
-        if isinstance(i, orm.DBResourceInput) and isinstance(bi, orm.DBResourceInput) and i == bi:
-            return (format_input(i), )
 
-        return (format_input(i), backtrack(bi))
+@resource.command()
+@click.argument('resource')
+@click.argument('input_name')
+def effective_iv(resource, input_name):
```
Comment (loles) [inline solar/solar/cli/resource.py]: iv? can you make it something more clear?

Comment (pigmej) [inline solar/solar/cli/resource.py]: iv => input value

Just wanted to make it shorter for easier typing.

Comment (zen) [inline solar/solar/cli/resource.py]: I agree with Łukasz, even effective_input_value is better.

Comment (pigmej) [inline solar/solar/cli/resource.py]: Changed, when https://github.com/Mirantis/solar/pull/280 will be merged, then shortcuts like this wouldn't make sense.

**Pull request merged**